packages/database/src/migrations/20201201003822-ConvertSurveyRawDataDownloadToCustomDataDownload-modifies-data.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+import { updateValues } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dataElementCode = 'RHFSC';
+const dashboardReportId = 'UNFPA_Raw_Data_Reproductive_Health_Facility';
+
+const previousDashboardReport = {
+  dataBuilder: 'rawDataDownload',
+  dataBuilderConfig: {
+    surveys: [
+      {
+        code: 'RHFSC',
+        name: 'Reproductive Health Facility Spot Check',
+      },
+    ],
+  },
+};
+
+const newDashboardReport = {
+  dataBuilder: 'surveyDataExport',
+  dataBuilderConfig: {
+    surveys: [
+      {
+        code: dataElementCode,
+        name: 'Reproductive Health Facility Spot Check',
+      },
+    ],
+    exportDataBuilder: {
+      dataBuilder: 'rawDataValues',
+      dataBuilderConfig: {
+        skipHeader: false,
+        surveysConfig: {
+          [dataElementCode]: {
+            entityAggregation: {
+              dataSourceEntityType: 'facility',
+            },
+          },
+        },
+        transformations: [
+          {
+            type: 'transposeMatrix',
+          },
+        ],
```

Comment:
This looks correct but I would like to ask @nick-beyondessential to place his `surveyDataExport` expert eyes on this

Reply:
Perfect!